src/test/java/com/jwplayer/southpaw/util/ByteArraySetTest.java

```diff
@@ -46,6 +74,655 @@ public ByteArraySet createSmallSet() {
         return set;
     }
 
+    public List<ByteArray> getRandomByteArrays(int count, long seed) {
+        System.out.println("Generating random " + count + " Byte Array(s) with " + seed + " seed");
```

Comment:
We should use logger instead of system.out. 

Like
```java
private static final Logger logger = LoggerFactory.getLogger(ByteArraySetTest.class);
```

Also I don't think we need to log this everytime the function is called. I'd suggest we only log out the random seed once before the whole class runs. The count is fixed in each test call, so the number itself isn't useful.

Can do a before hook like
```@Before
    public void setUp() {}
```

Reply:
Also whichever one remains can be made `private`

Reply:
There is no `BeforeAll` hook but I log it in the constructor